src/services/dumpers/dumper-v1.js

```diff
@@ -0,0 +1,377 @@
+const _ = require('lodash');
```

Comment:
Not a big fan of this name, if you have any suggestion, fire ! 

Something like agent-express-dumper.js ?

Reply:
`forest-express-dumper` ? Since it's for `forest-express-sequelize` and `forest-express-mongoose`

Reply:
Agreed
In the future this repo is very likely to support other agents (if we release standalone agents for all stacks)

I would call this `forest-express`, `agent-nodejs`, `forest-rails`, ...

Reply:
Will do in the last PR, so that the rebasing of other PRs doesn't become hell